code-quality/accessibility/WCAG-testing.md

```diff
@@ -0,0 +1,102 @@
+# WCAG Testing Procedure
+
+This is the procedure that has been used to test the FUSD WordPress sites
+
+## Automated testing
+
+There are 3 primary tools used in testing, they are all Chrome browser extensions:
+
+- [WAVE Browser Extension for Chrome and Firefox](https://wave.webaim.org/extension/)
+- [SiteImprove Browser Extension](https://siteimprove.com/en-us/core-platform/integrations/browser-extensions/)
+- [Broken Link Checker](https://chrome.google.com/webstore/detail/broken-link-checker/nibppfobembgfmejpjaaeocbogeonhch)
+
+The first two go over the site currently loaded in the browser and point out possible issues. The third will test all links on a site. The first two tests should be done on as many pages as a user is likely to encounter upon a visit. If you can check all of them, great, but if your site has 100 pages, and 95 of those are the result of a template, you don't need to check all 100.
```

Comment:
We may want to note that the user should not use this software on preview pages generated by site CMS (like WordPress).

Reply:
Ahhh, I forgot about that, good catch! I'll add that. Yeah, running these tests with the WP dashboard menu running is a real experience.